antora-ui-camel/src/css/vars.css

```diff
@@ -120,6 +120,7 @@
   --footer-font-color: var(--color-gray-70);
   --footer-link-font-color: var(--color-jet-80);
   /* dimensions */
+  --vw-base--desktop: calc(100 / 1300);   /* used to compute vw value from desired viewport size i.e. 1300 */
```

Comment:
Can I ask why 1300 is the desired viewport size? Is it limited by something else?

Reply:
No, the desired viewport size is not limited by anything. Just a personal coice.